docs/services/slam/_index.md

```diff
@@ -20,9 +20,9 @@ Breaking changes are likely to occur, and occur often.
 SLAM is an important area of ongoing research in robotics, particularly for mobile applications such as drones, boats, and rovers.
 
 The Viam SLAM service supports the integration of SLAM as a service on your robot.
-You can conduct SLAM with data collected live by a [RPlidar](/modular-resources/examples/rplidar/) or with LIDAR data you provide in configuration, and easily view the map you build on the **Control** tab of your robot's page in the [Viam app](https://app.viam.com):
+You can conduct SLAM with data collected live by a [RPlidar](/modular-resources/examples/rplidar/) or with LIDAR data you provide in configuration, and easily view the map you build on the **SLAM library** tab of your location's page in the [Viam app](https://app.viam.com):
```

Comment:
@EshaMaharishi Is this change ok/accurate? The old image had the old UI font and was Control tab view, figured this would be better. Got the map from Vijay!

Reply:
This looks fine! But would you mind using this image of the whole office map? https://drive.google.com/file/d/1E8gyd2svIIdQ-idON6Hv1rGv97EqoUhH/view?usp=drive_link

Reply:
@EshaMaharishi No problem, done!